### PR TITLE
Molim-56 bugfix/incorrect currency

### PR DESCRIPTION
**Pull request by tomas862**

### CODE REVIEW

File: src/Builder/InvoicePdfTemplateBuilder.php [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+        $orderFee = new MolOrderFee($orderFeeId);
+
+        if (!$orderFee->order_fee) {
+            return [];
+        }
+
+        return [
+            'orderFeeAmountDisplay' => Tools::displayPrice(
+                $orderFee->order_fee,
+                new Currency($this->order->id_currency)
```
Comment (tomas862) [inline src/Builder/InvoicePdfTemplateBuilder.php:46]: this is actually the place the whole PR is about - when displaying PDF its not wise to take currency from context. So a currency from order should be used instead

File: config/builder.yml [hunk 2]
```diff
@@ -16,3 +16,7 @@ services:
     arguments:
       - '@=service("mollie.factory.module_factory").getModuleVersion()'
       - '@Mollie\Service\ApiService'
+
```
Comment (tomas862) [inline config/builder.yml:19]: took a liberty to extract unwanted code from hook and put in template builder

**Review by margud — APPROVED** [review]
Looks good just make sure that all functions exist in ps1.6

**Comment by tomas862** [comment]
> Looks good just make sure that all functions exist in ps1.6

yeah, Tools::displayPrice and its params are for ages the same so no worries regarding this :)

**Review by lukvai — APPROVED** [review]

**Pull request merged**